servant-server/servant-server.cabal

```diff
@@ -52,6 +52,7 @@ library
       , network-uri        >= 2.6  && < 2.7
       , mtl                >= 2    && < 3
       , mmorph             >= 1
+      , network            >= 2.6
```

Comment:
I'm in favour of upper bounds too - we get reports via stackage as soon as a new version comes out, and between being certainly briefly not up on the latest version and possibly being broken and not knowing it, I'd take the former